tests/other_test.go

```diff
@@ -0,0 +1,39 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPlanValidation(t *testing.T) {
```

Comment:
why not add this to pr_test.go so we test in all PRs? There is no overhead as its just a plan test. 
Also your only testing `us-south` and `us-east`. How about wrapping it in a loop and testing all the supported / unsupported regions?

Reply:
I've added it to the PR test so it will be run every time, one possible issue that could arise from this is the plans for the invalid regions are outputting all of their error texts which could cause confusion when debugging issues. Should we modify the test to not output these plans and just print the output if the plan doesn't behave as expected?

Reply:
I've formatted the test so that it doesn't display the terraform plan output, but in looking for a way to suppress the validation error from showing up in the output I've found that we'd have to redirect the stdout of the test to /dev/null which isn't an acceptable solution so I've added a wrapper around the output of that error in the test to clarify that it's expected.